minisforum/v3/accelerometer.nix

```
@@ -0,0 +1,39 @@
+{ pkgs, ... }:
+{
+
+  # Override the DSDT ACPI table to fix the accelerometer.
+  # A driver already exists for a similar sensor.
+  # This overrides the IDs to make the existing driver work.
+  # From [Accelerometer on Linux](https://github.com/mudkipme/awesome-minisforum-v3/issues/2#issuecomment-2279282784)
```

```suggestion
  # From Accelerometer on Linux https://github.com/mudkipme/awesome-minisforum-v3/issues/2#issuecomment-2279282784
```
Those are not markdown 😓